assets/queries/terraform/aws/eks_cluster_encryption_disabled/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	cluster := input.document[i].resource.aws_eks_cluster[name]
+
+	not common_lib.valid_key(cluster, "encryption_config")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_eks_cluster[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'encryption_config' is defined and not null",
+		"keyActualValue": "'encryption_config' is undefined or null",
+		"searchLine": common_lib.build_search_line(["resource", "aws_eks_cluster", name], []),
+	}
+}
```

Comment:
I think we should add a verification that checks `resources` list contains `'secret'` as one of its values